src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPoolManager.cs

```diff
@@ -359,7 +359,7 @@ public ValueTask<HttpResponseMessage> SendProxyConnectAsync(HttpRequestMessage r
 
         public ValueTask<HttpResponseMessage> SendAsync(HttpRequestMessage request, bool async, bool doRequestAuth, CancellationToken cancellationToken)
         {
-            return HttpTelemetry.Log.IsEnabled() && request.RequestUri != null ?
+            return HttpTelemetry.Log.IsEnabled() && request.Version.Major < 3 && request.RequestUri != null ?
```

Comment:
Should there be a tracking issue and a link saying we need to remove this condition once we add telemetry for H3? Or what is the plan for H3?

Reply:
I opened a tracking issue for this: #40896

Reply:
Could you add comment here with the link, something like:
```
// [ActiveIssue("https://github.com/dotnet/runtime/issues/40896")]
```

Reply:
Added